coil-base/src/main/java/coil/ImageLoaderBuilder.kt

```diff
@@ -15,15 +15,16 @@ import coil.util.getDrawableCompat
 import kotlinx.coroutines.CoroutineDispatcher
 import kotlinx.coroutines.Dispatchers
 import okhttp3.Cache
+import okhttp3.Call
 import okhttp3.Dispatcher
 import okhttp3.OkHttpClient
 
 /** Builder for an [ImageLoader]. */
 @BuilderMarker
 class ImageLoaderBuilder(private val context: Context) {
 
-    private var okHttpClient: OkHttpClient? = null
-    private var okHttpClientBuilder: (OkHttpClient.Builder.() -> Unit)? = null
+    private var callFactory: Call.Factory? = null
+    private var canRebuildWithOptimizations: Boolean = false
```

Comment:
Let's remove `canRebuildWithOptimizations` since we should only apply these optimizations if we need to build the default call factory.

Reply:
By happy coincidence, I forgot to wire this up anywhere. Just had to delete the field. 34b0090